WcaOnRails/db/migrate/20160502113530_add_preferred_events_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferredEventsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :preferred_event_ids, :text
```

Comment:
Hmm, sorry but I don't see it. How would this update the PreferredEvents table, is it some Rails magic? Also could you explain these private things a bit more.